frontend/src/pages/ranking/index.tsx

```diff
@@ -32,7 +35,15 @@ export const RankingPage: React.SFC = () => {
   });
 
   const [filters, { select }] = useFormState();
-
+  const filterVisibleSubmissions = (submission) => {
```

Comment:
we can keep this as it is but when we work on the voting improvements (i.e server-side filtering) we should migrate  update this page too